.gitlab-ci.yml

```diff
@@ -2580,3 +2631,31 @@ stages:
     JANE_INVENTORY_HOSTVARS: 'yadm__dotfiles_enabled=true'
     JANE_DIFF_PATTERN: '.*/roles/yadm/.*'
     JANE_LOG_PATTERN: '\[yadm\]'
+
+
+# === Ansible no dependencies roles === [[[2
+
+'ferm role':
+  extends: .tpl_molecule_no_deps
+  needs:
+    - 'bootstrap playbook'
+
```

Comment:
I can't add rules for now as this would generate errors in the case ferm was not run.